okapi-core/src/main/java/org/folio/okapi/managers/InternalModule.java

```diff
@@ -330,6 +337,30 @@ public static ModuleDescriptor moduleDescriptor(String okapiVersion) {
         + "    \"permissionsRequired\" : [  \"okapi.proxy.tenants.interfaces.get\" ], "
         + "    \"permissionsRequiredTenant\" : [ ], "
         + "    \"type\" : \"internal\" "
+        + "   }, {"
+        + "    \"methods\" :  [ \"GET\" ],"
+        + "    \"pathPattern\" : \"/_/proxy/tenants/{tenantId}/timers\","
+        + "    \"permissionsRequired\" : [  \"okapi.proxy.tenants.timers.list\" ], "
+        + "    \"permissionsRequiredTenant\" : [ ], "
+        + "    \"type\" : \"internal\" "
+        + "   }, {"
+        + "    \"methods\" :  [ \"PATCH\" ],"
+        + "    \"pathPattern\" : \"/_/proxy/tenants/{tenantId}/timers\","
+        + "    \"permissionsRequired\" : [  \"okapi.proxy.tenants.timers.patch\" ], "
+        + "    \"permissionsRequiredTenant\" : [ \"okapi.proxy.self.timers.patch\" ], "
+        + "    \"type\" : \"internal\" "
+        + "   }, {"
+        + "    \"methods\" :  [ \"GET\" ],"
+        + "    \"pathPattern\" : \"/_/proxy/tenants/{tenantId}/timers/{timerId}\","
+        + "    \"permissionsRequired\" : [ \"okapi.proxy.tenants.timers.get\" ], "
+        + "    \"permissionsRequiredTenant\" : [ ], "
+        + "    \"type\" : \"internal\" "
+        + "   }, {"
+        + "    \"methods\" :  [ \"PATCH\" ],"
+        + "    \"pathPattern\" : \"/_/proxy/tenants/{tenantId}/timers/{timerId}\","
+        + "    \"permissionsRequired\" : [  \"okapi.proxy.tenants.timers.patch\" ], "
+        + "    \"permissionsRequiredTenant\" : [ \"okapi.proxy.self.timers.patch\" ], "
```

Comment:
Is `okapi.proxy.self.timers.patch` defined?

Reply:
Good.  Will add.